libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/navigationview/NavigationViewListener.kt

```diff
@@ -141,13 +141,6 @@ abstract class NavigationViewListener {
      */
     open fun onInfoPanelSlide(slideOffset: Float) = Unit
 
-    /**
-     * Called when hardware back button has been pressed.
-     *
-     * @return True if the listener has consumed the event, false otherwise.
-     */
-    open fun onBackPressed(): Boolean = false
```

Comment:
Let's include migration instructions in the CHANGELOG entry.

Reply:
I have already mentioned what should be used instead of `onBackPressed`, isn't that enough?

Reply:
I think it's nice to provide a short code snippet that shows how to patch old usage. I always found it extremely helpful when upgrading the library versions.

Reply:
I added a link to the documentation at https://developer.android.com. It is hard to provide a short migration guide, because the two approaches are not exactly equivalent. Previously we had to do runtime checks in the callback and return false if we don't want to handle the event, but now we have to disable and enable the callback when necessary.